.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           command: sh ./scripts/create-placeholder.sh
       - run:
           name: Install dependencies
-          command: yarn --frozen-lockfile --link-duplicates --network-concurrency 1
+          command: yarn --frozen-lockfile --network-concurrency 1
```

Comment:
Removing this option fixes an issue I ran into after bumping the hardhat version. The option is [known to be unstable](https://github.com/yarnpkg/yarn/issues/2734) and doesn't add much value in our case.

For context, this option is generally used to reduce the size of the installed dependencies in situations where that would help with performance such as when working on a hosted frontend.